build/tools/checkout.py

```diff
@@ -94,7 +110,11 @@ def checkout_repo(cwd, repo):
     repo_path = repo['path']
     repo_url = repo['url']
     mirror_url = e(f'${{REPO_{repo_name.replace("-", "_").upper()}_URL}}')
-    branch = repo['branch']
```

Comment:
Wouldnt it be easier/simpler to do:
branch = e(f'${{REPO_{repo_name.replace("-", "_").upper()}_BRANCH}}') or repo['branch']

Probably no other changes would be necessary at all

Reply:
No, because I have no idea what e is, I checked python utils and there is no e module in utils.
```
In [1]: from utils import e                                                                          
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-550952e6e02e> in <module>
----> 1 from utils import e

ImportError: cannot import name 'e' from 'utils' (/usr/local/lib/python3.7/site-packages/utils/__init__.py)
```
The idea is to grab the branch parameter from Jenkins Custom build if the variable exists.

Reply:
Its used above, for mirror_url :)

e() is a custom method of build system, not python module. It will grab the environment variable.